services/age_service.js

```diff
@@ -0,0 +1,37 @@
+const ValidationError = require(`../helpers/ValidationError.js`);
+
+//gets the dob request, calculate and return age
+const calaculateAge = (dob) => {
+  //check if dob is passed
+  if (!dob) throw new ValidationError(`Please pass date of birth to continue`);
+  // try pass dob
+  let dobformated = new Date(dob);
+  //check if dob passed is string date format
+  if (dobformated == `Invalid Date`) {
+    // convert dob to number
+    const dobinnumber = Number(dob);
+    // check if number is less than 0
+    if(dobinnumber < 1) 
+      throw new ValidationError(`dob cannot be lesser than 1`); 
```

Comment:
Codacy has a fix for the issue: [Delete `·`](https://app.codacy.com/gh/phiiileo/validate-age/pullRequest?prid=9953536)
```suggestion
      throw new ValidationError(`dob cannot be lesser than 1`);
```